scene/2d/camera_2d.cpp

```diff
@@ -695,6 +736,15 @@ void Camera2D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_enable_follow_smoothing", "follow_smoothing"), &Camera2D::set_enable_follow_smoothing);
 	ClassDB::bind_method(D_METHOD("is_follow_smoothing_enabled"), &Camera2D::is_follow_smoothing_enabled);
 
+	ClassDB::bind_method(D_METHOD("set_enable_snapping", "snapping_enabled"), &Camera2D::set_enable_snapping);
```

Comment:
For the bindings of the 3 setters, I'd remove the `snapping_` part of the argument names, it's quite redundant with the method names themselves.